code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -500,6 +500,7 @@ GLOBAL_LIST_INIT(plastic_recipes, list(
 	new /datum/stack_recipe("plastic flaps", /obj/structure/plasticflaps, 5, one_per_turf = TRUE, on_floor = TRUE, time = 40), \
 	new /datum/stack_recipe("water bottle", /obj/item/reagent_containers/glass/beaker/waterbottle/empty), \
 	new /datum/stack_recipe("large water bottle", /obj/item/reagent_containers/glass/beaker/waterbottle/large/empty,3), \
+	new /datum/stack_recipe("pill bottle", /obj/item/storage/pill_bottle,2), \
```

Comment:
personally I think this would be better in a autolathe than a crafting recipe on a stack.